pyerrors/fits.py

```diff
@@ -151,6 +151,12 @@ def func_b(a, x):
         For details about how the covariance matrix is estimated see `pyerrors.obs.covariance`.
         In practice the correlation matrix is Cholesky decomposed and inverted (instead of the covariance matrix).
         This procedure should be numerically more stable as the correlation matrix is typically better conditioned (Jacobi preconditioning).
+    corr_matrix: array, optional
+        if correlated_fit=True is set as well, can provide a correlation matrix (without y errros!) of your own choosing for a correlated fit
+    inv_chol_cov_matrix
+        if correlated_fit=True is set as well, can provide an inverse covariance matrix (y errros, dy_f included!) of your own choosing for a correlated fit
+        The matrix must be a lower triangular matrix constructed from a Cholesky decomposed correlation matrix (e.g. chol = np.linalg.cholesky(corr)
+        so that the inverse covariance matrix is defined by chol_inv = scipy.linalg.solve_triangular(chol, covdiag, lower=True) with covdiag = np.diag(1 / np.asarray(dy_f)))
```

Comment:
This is connected to the comment above: The current description is very hard to understand if the user does not look into the code but just uses package because some variables like `dy_f` are not introduced.